jekyll/_cci2/first-steps.md

```diff
@@ -75,6 +75,10 @@ To run your very first build on CircleCI, go to the [Sign Up](https://circleci.c
     - Connect to your GitHub or Bitbucket account to build and deploy your projects on CircleCI. Type your GitHub or Bitbucket username, password, and two-factor authorization if applicable, then click Sign In/Login.
     - Explore the app using a popular open source project building on CircleCI ([React by Facebook](https://app.circleci.com/pipelines/github/facebook/react)), or one of our own sample projects: a [sample JavaScript app](https://app.circleci.com/pipelines/github/CircleCI-Public/sample-javascript-cfd/), and a [sample Python app](https://app.circleci.com/pipelines/github/CircleCI-Public/sample-python-cfd/). You'll be able to start exploring features such as [pipelines]({{ site.baseurl }}/2.0/pipelines/) and [workflows]({{ site.baseurl }}). The Dashboard, Projects, Organization Settings, and Plan pages are not available until you connect your GitHub or Bitbucket accounts.
 
+## Terms
+{: #terms}
+
+By signing up, you are agreeing to our SaaS Agreement and Privacy Policy. We ask for read/write access to make your experience seamless on CircleCI. If you are a GitHub user and aren’t ready to share access to your private projects, you can choose public repos instead. Protected by reCAPTCHA, Google Privacy Policy and Terms of Service apply.
```

Comment:
There are no links here? If we are referencing legal documents, I would expect we need to link out to them. Specifically the SaaS Agreement, Privacy Policy, Google Privacy Policy, and Terms of Service.